pandas/core/_numba/kernels.py

```diff
@@ -0,0 +1,113 @@
+"""
```

Comment:
make kernels.py import each of the kernels themsleves and expose in this namespace